data/pirate jobs.txt

```diff
@@ -2465,6 +2465,58 @@ mission "Cargo Theft (South)"
 		payment 1500000
 		dialog "The pirates of <planet> take the secret cargo from your cargo hold and pay you <payment>."
 
+mission "Capture Smuggler"
+	name "Capture rogue smuggler"
+	minor
+	description `A smuggler absconded with his illegal cargo near <system>. The unknown person he stole it from wants it back. Recover the cargo from the <npc> and return both it and the smuggler to <origin> for <payment>.`
+	to offer
+		random < 15
+		"combat rating" > 2000
+		"reputation: Pirate" > 10
+	source
+		government "Pirate"
+	npc board
+		conversation
+			`After neutralizing the crew and capturing the captain, you have your men sweep the ship. Ultimately they find the stolen cargo in the captain's cabin, hidden behind a false panel. The cargo is a nondescript storage container, but upon closer inspection you see that it's code-locked and covered with interference plating.`
+				launch
+		government "Merchant"
+		personality staying uninterested timid target marked
+		system
+			distance 2 3
+		fleet
+			names "civilian"
+			variant
+				"Modified Argosy (Smuggler)"
+	on fail
+		dialog "You have failed to capture the smuggler and recover the stolen cargo. This will probably not be good for your reputation."
+		"reputation: Pirate" <?= -10
+	on offer
+		conversation
+			`As you step into a dive bar for a quick drink, the largest man you've ever seen stands up and blocks your path. "<first> <last>," he says in a thick accent. "My employer hears you do certain jobs, take money, do not ask questions. He hears you are reliable. A man, a smuggler, he steals cargo from my employer. My employer very much wants cargo returned, and this man as well."`
```

Comment:
Can we define large here? Tall? Muscular? Fat? All three things at the same time? This could be left as is, I'm just somewhat interested in the details, because this sounds like quite the character.